src/index.html

```diff
@@ -11,7 +11,226 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="form-block">
+
+        <legend>Personal information</legend>
+        <div class="block-input">
+          <label for="surname">
+            Surname:
+          </label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="block-input">
+          <label for="name">
+            Name:
+          </label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </div>
+
+
+        <div class="block-input">
+          <label for="year">
```

Comment:
one blank line maximum

```suggestion
        </div>

        <div class="block-input">
          <label for="year">
```